scripts/batch-dqac-conversion-script.py

```diff
@@ -0,0 +1,79 @@
+import requests
+import yaml
+import os
+import time
+import json
+
+API_URL = "https://qa.siffletdata.com/api/v1/rules/_all-as-code"
+OUTPUT_DIR = "monitors_as_code"
+
+os.makedirs(OUTPUT_DIR, exist_ok=True)
+
+headers = {
+    "Authorization": "Bearer <Token>",
+    "Accept": "application/x-yaml"
+}
```

Comment:
```suggestion
TENANT = "<target tenant>"
TOKEN = "<your token>"

API_URL = f"https://{TENANT}.siffletdata.com/api/v1/rules/_all-as-code"
OUTPUT_DIR = "monitors_as_code"

os.makedirs(OUTPUT_DIR, exist_ok=True)

headers = {
    "Authorization": f"Bearer {TOKEN}",
    "Accept": "application/x-yaml"
}
```

We should help the user with placeholder variables